### DISCUSSION TITLE
v.0.1.0.2023-01-16

### DISCUSSION THREAD

**Discussion by engeniousio** [engeniousio/sift#28] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/engeniousio/sift-ios/releases/tag/v.0.1.0.2023-01-16'>v.0.1.0.2023-01-16</a>.</em>

**Comment by engeniousio** [comment]
test

**Comment by o-vashenko-engenious** [comment]
Hi! Happy to announce that we've updated the manuals for SIFT. Many new details were added to simplify the installation and config processes.
Also, the documentation was supplemented with videos.
You can find the documentation in the Wiki section, or by the [link](https://github.com/engeniousio/sift-ios/wiki/SIFT-iOS-Local-Set-Up)
